File: practice19/1.cpp
//https://www.nowcoder.com/practice/967133d6656440ba951870eaf17861de?tpId=98&tqId=32858&tPage=2&rp=2&ru=/ta/2019test&qru=/ta/2019test/question-ranking
#include<iostream>
using namespace std;
int main()
{
	int i, j, m, n, k, sign;
	while (cin >> n)
	{
		m = 1;
		sign = 0;
		for (i = 1; i <= n;)
		{
			k = i;
			for (j = 1; j <= m; j++)
			{
				i++;
				if (i > n)
				{
					sign = 1;
					break;
				}
			}
			if (sign == 1)
			{
				break;
			}
			m++;
		}
		cout << m << endl;
	}
	system("pause");
	return 0;
}

File: practice31/1.java

//https://www.nowcoder.com/practice/e90351c561ff40a4b2bbf067bbbb5f31?tpId=90&tqId=30859&tPage=5&rp=5&ru=/ta/2018test&qru=/ta/2018test/question-ranking
import java.util.Scanner;

public class Main{
    public static void main(String args[]){
        Scanner scanner=new Scanner(System.in);
        int n=scanner.nextInt();
        int m=scanner.nextInt();
        if(n<=m){
            System.out.println(2);
        }
        else{
            int x= (int) Math.ceil((double)n/m*2);
            System.out.println(x);
        }
    }
}